text/0000-erfc-post-build-contexts.md

```diff
@@ -373,27 +366,27 @@ to standardize things like json output and whatnot.
 
 ## Namespacing
 
-Currently, two post-build contexts can both declare interest in the same
+Currently, two frameworks can both declare interest in the same
 attributes. How do we deal with collisions (e.g., most test crates will
 want the attribute `#[test]`). Do we namespace the attributes by the
-context name (e.g., `#[mytest::test]`)? Do we require them to be behind
+framework name (e.g., `#[mytest::test]`)? Do we require them to be behind
 `#[cfg(mytest)]`?
 
 ## Runtime dependencies and flags
 
-The code generated by the post-build context may itself have dependencies.
-Currently there's no way for the post-build context to specify this. One
+The code generated by the framework may itself have dependencies.
+Currently there's no way for the framework to specify this. One
 proposal is for the crate to specify  _runtime_ dependencies of the
-post-build context via:
+framework via:
 
 ```toml
-[context-dependencies]
+[testing.framework.dependencies]
```

Comment:
I'm weary of scoping these dependencies under `testing.framework`, as that's the key used by crates that *use* a CTF.